spec/helpers/application_helper_spec.rb

```diff
@@ -151,4 +151,24 @@
       expect(result).to have_link(text: /articles/, href: '/articles?q=my+query&search_field=title')
     end
   end
+
+  # TODO: Add bento search link
+  describe '#link_to_bento_search' do
+    subject(:result) { Capybara.string(helper.link_to_bento_search) }
+    xit 'passes query to bento search params' do
+      params[:q] = 'my query'
+      expect(result).to have_link(text: /all/ )
+    end
+  end
+
+  describe '#link_to_library_search' do
+    subject(:result) { Capybara.string(helper.link_to_library_website_search) }
+    before do
+      allow(helper).to receive(:blacklight_config).and_return(double(index: double(search_field_mapping: { search_title: :title })))
+    end
+    it 'passes query to library website search params and does not pass search fields' do
+      params[:q] = 'kittens'
+      expect(result).to have_link(text: /library website/)
```

Comment:
I'm having trouble testing an `href` value for this link.
`expect(result).to have_link(text: /library website/, href: 'https://library.stanford.edu/search/all?search=kittens')`

I keep getting a `result` with a root path, but the method is working the application. Maybe I'm missing something in my test setup?

Reply:
@camillevilla hmm, that is odd.  What is the helper method's value when running this test?

Reply:
Have you tried `controller.params = { q: 'kittens' }`?

Reply:
sweet. That worked!